treeherder/webapp/api/runnable_jobs.py

```diff
@@ -19,7 +19,7 @@ def list(self, request, project):
         """
         GET method implementation for list of all runnable buildbot jobs
         """
-        decision_task_id = request.query_params.get('decisionTaskID')
+        decision_task_id = request.query_params.get('decision_task_id')
```

Comment:
So you will need to make the corresponding change in the JS code as well. 
https://github.com/mozilla/treeherder/blob/f7b3d2f67d218580d8d5be97d558c5e72e57360d/ui/js/models/resultset.js#L205